db/models.py

```diff
@@ -1 +1,31 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             related_name="races",
+                             on_delete=models.CASCADE)
+    guild = models.ForeignKey(Guild,
+                              null=True,
+                              related_name="guilds",
+                              on_delete=models.SET_NULL)
```

Comment:
related names should be as class name in plural

Reply:
Oh, yes...